Index.html

```diff
@@ -0,0 +1,37 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio-Setup-and mobile-first</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <header>
+    <nav class="bar">
+    <a class="name">Susana González</a>
+    <img class="hamburguer"src="./images/hamburguer.svg">
+</nav>
+  </header>
+  <main>
+    <section class="hello">
+      <h1 class="titulo">Hey There, I&apos;m Susan!</h1>
+      <h2 class="subtitulo">I&apos;am a Software Developer</h2>
+      <p class="parrafo">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don&apos;t hestiate to contact me.</p>
+        <div class="grid-container">
+          <div class="grig-github"href="https://github.com/Zuz4n/"><img class="github"src="./images/Github.svg"></div>
+          <div class="grig-linkedin"href="www.linkedin.com/in/susana-glez-gdo"><img class="linkedin"src="./images/Linkedin.svg"></div>
+          <img class="newent"src="./images/new.svg">
+          <div class="grid-twitter"href="https://twitter.com/SuschenSu"><img class="twitter"src="./images/Twitter.svg"></div>
+          <div class="grid-gmail"href="siuxmicroverse@gmail.com"><img class="mail"src="./images/Mail.svg"></div>
+        </div>
```

Comment:
- [ ] As this represents a list of social media icons, It would be better to use the `<ul>` tag instead of `<div>` as requested by the previous reviewer.  The `<ul>` tag is used to describe an unordered list 📜 , so it not only improves the semantic value of your project, but it also helps accessibility tools such as screen readers to understand the structure of your page. Providing a better experience for users that rely on this kind of technology. 🤓 

Gabriel provided an excellent example that you could adapt in order to apply this requirement:

```Html
<ul>
  <li><a href="your social media URL">the social media icon 1</a></li>
  <li><a href="your social media URL">the social media icon 2</a></li>
  ...
</ul
```

Please, change this. 🛠️